neqo-http3/src/connection_client.rs

```diff
@@ -689,15 +689,20 @@ impl Http3Client {
     }
 
     #[must_use]
-    pub fn qpack_decoder_stats(&self) -> &Stats {
+    pub fn qpack_decoder_stats(&self) -> QStats {
         self.base_handler.qpack_decoder.stats()
     }
 
     #[must_use]
-    pub fn qpack_encoder_stats(&self) -> &Stats {
+    pub fn qpack_encoder_stats(&self) -> QStats {
         self.base_handler.qpack_encoder.stats()
     }
 
+    #[must_use]
+    pub fn transport_stats(&self) -> TStats {
+        self.conn.stats()
+    }
```

Comment:
I think that it would be better overall to have a single stats object for the HTTP layer that bundles up all of the stats.  That means a larger clone, but I think that it would be easier to use.

Reply:
I will leave it like this for now. For use with necko there is no much difference.